src/main/java/umk/mat/pajda/ProjektZespolowy/entity/Review.java

```diff
@@ -1,35 +1,39 @@
 package umk.mat.pajda.ProjektZespolowy.entity;
 
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.Id;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.FutureOrPresent;
 import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
 import java.time.LocalDateTime;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
-import lombok.ToString;
 
 @Entity
 @Setter
 @Getter
 @NoArgsConstructor
-@ToString
 @AllArgsConstructor
 public class Review {
 
   @Id @GeneratedValue @NotNull private Integer id;
 
+  @Size(min = 0, max = 10)
```

Comment:
default value of min is 0, so its not necessary

Reply:
you're right